### ISSUE TITLE
Error: custom keyword definition is invalid: data.errors should be boolean

### ISSUE THREAD

**Issue by simsketch** [epoberezkin/ajv-errors#27]
I ran `vue create client` but when i try `npm run serve` I get the error below.

```$ vue-cli-service build --mode production --dest dist --target app --dashboard


-  Building for production...

 ERROR  Error: custom keyword definition is invalid: data.errors should be boolean
Error: custom keyword definition is invalid: data.errors should be boolean
    at Ajv.addKeyword (/Users/bocaops/Code/fullstack_vue_express/client/node_modules/ajv/lib/keyword.js:65:13)
    at module.exports (/Users/bocaops/Code/fullstack_vue_express/client/node_modules/ajv-errors/index.js:10:7)
    at Object.<anonymous> (/Users/bocaops/Code/fullstack_vue_express/client/node_modules/schema-utils/src/validateOptions.js:22:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)

Total task duration: 1.1s```

**Comment by DBosley** [comment]
Same as #26

**Comment by simsketch** [comment]
Thanks, I followed his advice and changed line `14` of `ajv-errors/index.js` to `errors: true` instead of `errors: 'full'`

**Comment by epoberezkin** [comment]
'full' is correct value, it's fixed in ajv 6.9.1
